powerlevel9k.zsh-theme

```diff
@@ -156,12 +156,12 @@ left_prompt_segment() {
   if [[ -n $6 ]]; then
     visual_identifier="$(print_icon $6)"
     if [[ -n "$visual_identifier" ]]; then
+      # Add an whitespace if we print more than just the visual identifier
+      [[ -n "$5" ]] && visual_identifier="$visual_identifier "
```

Comment:
Could you add a comment why there is a whitespace before ending the foreground color?

Reply:
Done.